tracer/test/benchmarks/Benchmarks.Trace/SerilogBenchmark.cs

```diff
@@ -56,40 +52,17 @@ public void EnrichedLog()
             {
                 using (LogInjectionTracer.StartActive("Child"))
                 {
-                    for (int i = 0; i < N; i++)
-                    {
-                        var logEvent = new LogEvent(
-                            DateTimeOffset.Now,
-                            LogEventLevel.Information,
-                            exception: null,
-                            new MessageTemplate("Hello", Enumerable.Empty<MessageTemplateToken>()),
-                            properties: Enumerable.Empty<LogEventProperty>());
-
-                        // equivalent of auto-instrumentation
-                        var tracer = LogInjectionTracer;
-                        var dict = logEvent.DuckCast<LogEventProxy>().Properties;
-                        AddPropertyIfAbsent(dict, CorrelationIdentifier.SerilogServiceKey, tracer.DefaultServiceName);
-                        AddPropertyIfAbsent(dict, CorrelationIdentifier.SerilogVersionKey, tracer.Settings.ServiceVersion);
-                        AddPropertyIfAbsent(dict, CorrelationIdentifier.SerilogEnvKey, tracer.Settings.Environment);
+                    var logEvent = new LogEvent(
+                        DateTimeOffset.Now,
```

Comment:
Can we also initialize this value before all test runs? I'm not sure how much it will reduce variance but it may be worth doing

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2204/commits/8d5327734066da79a029e873163f788a3998182d